azure-mgmt-storage/setup.py

```diff
@@ -78,7 +78,7 @@
     packages=find_packages(exclude=["tests"]),
     install_requires=[
         'msrestazure>=0.4.27,<2.0.0',
-        'azure-common~=1.1',
+        'azure-common~=1.1,>=1.1.9',
```

Comment:
Please remove this line that I already fixed on master and I will merge. Thanks!